pkg/kubelet/cadvisor.go

```diff
@@ -59,7 +59,7 @@ func (kl *Kubelet) GetContainerInfo(podFullName string, uid types.UID, container
 	if cc == nil {
 		return nil, nil
 	}
-	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.dockerClient, false)
+	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.containerRuntime, false)
```

Comment:
@derekwaynecarr They don't have to as we can provide different cmd options to different kubelet. But I am not seeing reasons why we need different runtime in the same cluster. Maybe for live updating the cluster?